{
 "cells": [
  {
   "cell_type": "markdown",
   "id": "cd919842-1132-4711-9e22-a81c3a039114",
   "metadata": {},
   "source": [
    "Conectar ao MongoDB"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "a30e5df3-bdf6-4477-bff5-dc5bd4d99339",
   "metadata": {},
   "outputs": [],
   "source": [
    "pip install pymongo"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "b041461c-2582-40c5-8c3a-27d98932a02e",
   "metadata": {},
   "outputs": [],
   "source": [
    "from pymongo import MongoClient\n",
    "from pymongo.errors import ConnectionFailure\n",
    "print(\"Conectando ...\")\n",
    "try:\n",
    "    client = MongoClient(\"mongodb://root:mongo@172.24.0.2:27017/\", serverSelectionTimeoutMS=5000)\n",
    "    client.server_info()  # Isso lançará uma exceção se não puder se conectar ao servidor.\n",
    "    print(\"Conexão estabelecida com sucesso!\")\n",
    "\n",
    "except ConnectionFailure:\n",
    "    print(\"Falha na conexão ao servidor MongoDB\")"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
